src/Validator/Rules/KnownTypeNames.php

```diff
@@ -65,4 +109,14 @@ public static function unknownTypeMessage(string $type, array $suggestedTypes):
 
         return $message;
     }
+
+    /**
+     * @param Node|array<int, Node> $value
+     */
+    public static function isSDLNode($value): bool
+    {
+        return $value instanceof Node
+        // TODO: should be (TypeSystemDefinitionNode || TypeSystemExtensionNode)
```

Comment:
Because the PHP interfaces don't match JS code / GraphQL spec.

https://github.com/webonyx/graphql-php/blob/130ae955dd11e1efa52f526dc30d5a06ac17df84/src/Language/AST/TypeSystemDefinitionNode.php#L6-L10 vs https://github.com/graphql/graphql-js/blob/e91f3a88bacecd1e656132ea1bb6786999931058/src/language/ast.ts#L544-L547

Reply:
`TypeSystemExtensionNode` doesn't exist in this project. And `TypeExtensionNode` interface shoudn't extend `TypeSystemDefinitionNode`.

Reply:
Please review https://github.com/webonyx/graphql-php/pull/1073

Reply:
Will do tomorrow.